hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/HoodieTimelineArchiver.java

```diff
@@ -632,8 +629,9 @@ private void writeToFile(Schema wrapperSchema, List<IndexedRecord> records) thro
     }
   }
 
-  private IndexedRecord convertToAvroRecord(HoodieInstant hoodieInstant)
+  private IndexedRecord convertToAvroRecord(HoodieInstant hoodieInstant, boolean isEmpty)
```

Comment:
nit: given that the logic for empty instant is completely different, it's better to create a new method `createAvroRecordFromEmptyInstant()`.

Reply:
👍